apidoc/Titanium/Media/Media.yml

```diff
@@ -364,7 +364,7 @@ methods:
     platforms: [iphone, ipad]
     deprecated:
         since: "5.1.0"
-        notes: Use <Titanium.Media.requestAudioPermissions> instead.
+        notes: Use <Titanium.Media.requestRecorderPermission> instead.
```

Comment:
`requestRecorderPermission ` doesn't sound clear to me - it could be the video recorder as well. Use `requestAudioRecorderPermissions` instead.